5_People/career.md

```diff
@@ -1,4 +1,6 @@
-# Niteo Salary System
+# Career in Niteo
```

Comment:
Maybe Niteo Career? Career in Niteo sounds off for me. No strong opinion though.

Usually the prep _in_ is used when referring to a field (e.g. career in Science), and then we use the prep _at_ when it alludes to a specific location (works at Google). Again, no strong opinion.